sources/packages/web/src/types/contracts/StudentAppealContract.ts

```diff
@@ -20,5 +20,5 @@ export interface StudentAppealRequest {
   data: any;
   formName: string;
   approval?: StudentAppealApproval;
-  files?: string[];
+  files: string[];
```

Comment:
For the appeals typing files as optional seems to be right. But I see that DTOs are typed with files as required. Not a blocker here.